config/buildspec-dev.yml

```diff
@@ -3,9 +3,10 @@ version: 0.2
 env: 
   variables:
     STAGE: dev
-    SERVICE:
+    SERVICE: exampleservice
     PROJECT: MozDef-EF
-    API_PATH:
+    API_PATH: /example/event/api
+    TOKEN_ARN: "arn:aws:ssm:<REGION_NAME>:<ACCOUNT_ID>:parameter/<parameter-name>"
```

Comment:
I think this is a neat idea, but looking at `serverless.yml` as it is, I don't think we actually need to specify the full arn here as a variable. I will explain more in my comment below (in `serverless.yml`).